From be62aeee7a5abc55e04f2e985730f1efa73e98d5 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Mon, 9 Mar 2015 14:04:09 +0800
Subject: [PATCH 0214/1691] MLK-10387-2 ARM: imx7d-sdb: add USB support

commit be62aeee7a5abc55e04f2e985730f1efa73e98d5 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Add USB support for both USBOTG1 and USBOTG2.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
(cherry picked from commit 8b6ae9b5ca581dce29ccfc0640bf020bba5b55b4)
---
 arch/arm/boot/dts/imx7d-sdb.dts | 59 +++++++++++++++++++++++++++++++++++++++++
 1 file changed, 59 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 635d5e3..91faeef 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -398,6 +433,19 @@
 	status = "okay";
 };
 
+&iomuxc_lpsr {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_hog_2>;
+
+	imx7d-sdb {
+		pinctrl_hog_2: hoggrp-2 {
+			fsl,pins = <
+				MX7D_PAD_GPIO1_IO05__GPIO5_IO5	0x14
+			>;
+		};
+	};
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1>;
-- 
1.9.1

